### ISSUE TITLE
Not working with Rails > 6.0.0.rc1

### ISSUE THREAD

**Issue by louim** [enriclluelles/route_translator#195]
Hey!

We are using the rails `6-0-stable` branch while waiting for Rails to cut another Release candidate.

This PR https://github.com/rails/rails/pull/34656 has recently been merged to rails master and backported to the rails `6-0-stable` branch. It added an extra argument to the `Mapper::Mapping.new` https://github.com/rails/rails/commit/85855d63fc84c413b3e277d745691adccc5120c1#diff-2cfaf53c860732fea8689d6f2002594bR60 which makes it incompatible with the current version of route_translator here: https://github.com/enriclluelles/route_translator/blob/efcbf87b96453b350f925d6c102a80e450584fb7/lib/route_translator/extensions/route_set.rb#L39

I would be happy to submit a patch, but i'm not quite sure how you would like to handle the case to keep backward compatibility.

To summarize:
- Rails <= 6.0.0.rc1 : [14 arguments](https://github.com/rails/rails/blob/3054e1d584e7eca110c69a1f8423f2e0866abbf9/actionpack/lib/action_dispatch/routing/mapper.rb#L102)
- Rails > 6.0.0.rc1, < 6.1.0: [15 arguments](https://github.com/rails/rails/blob/1f41f2ac6dc659be9aabcd4cc096b31aa150eb2b/actionpack/lib/action_dispatch/routing/mapper.rb#L114)
- Rails master (6.1.0): [11 positional params](https://github.com/rails/rails/blob/2fe20cb55c76e6e50ec3a4ec5b03bbb65adba290/actionpack/lib/action_dispatch/routing/mapper.rb#L118)

**Comment by tagliala** [comment]
😩

I'm going to wait for rc2 release

**Comment by tagliala** [comment]
@louim I've found another way to implement this

Could you please giva a try to the [rails-gt-6.0.0.rc1-compatibility branch](https://github.com/enriclluelles/route_translator/tree/feature/rails-gt-6.0.0.rc1-compatibility)?

**Comment by louim** [comment]
@tagliala I can confirm it's working as expected with the rails 6 stable branch!

**Comment by louim** [comment]
rc2 is now out 🎉, @tagliala were you planning to merge https://github.com/enriclluelles/route_translator/pull/197 in the master branch?

**Comment by tagliala** [comment]
@louim 

This is potentially a breaking change if someone did some monkey patching, so I'm going to release a new major version,

I would like to merge also #200 and #198.

I was waiting for an answer on both those branches but I didn't get one

Anyway... The only required change for a standard usage of this gem with both #198 and #200 merged, is to explicitly add `around_action :set_locale_from_url` to the `ApplicationController`

If you can also run your application specs against 
- https://github.com/enriclluelles/route_translator/tree/feature/do-not-raise-invalid-locale-errors (there should be no breaking changes)

and

- https://github.com/enriclluelles/route_translator/tree/feature/explicitly-add-around-action (add `around_action :set_locale_from_url` to the `ApplicationController`)

I would be happy to merge everything and release version7 of route translator

**Comment by louim** [comment]
@tagliala I merged both PR locally on the develop branch https://github.com/hooktstudios/route_translator/tree/develop and tested with our app. I saw no problems. We were already skipping the `around_action` , I was able to remove the `skip_around_action` in our app and confirm that the application still worked correctly.

**Comment by tagliala** [comment]
@louim thanks for the heads-up

I will release a new version in the next few hours

**Comment by tagliala** [comment]
7.0.0 released
